src/config/feature-flags.ts

```diff
@@ -0,0 +1,50 @@
+import LaunchDarkly, { LDUser } from "launchdarkly-node-server-sdk";
+import { getLogger } from "./logger";
+import envVars from "./env";
+import crypto from "crypto";
+
+const logger = getLogger("feature-flags");
+
+const launchdarklyClient = LaunchDarkly.init(envVars.LAUNCHDARKLY_KEY, {
+	logger,
+	offline: !envVars.LAUNCHDARKLY_KEY
+});
+
+export enum BooleanFlags {
+	MAINTENANCE_MODE = "maintenance-mode",
+}
+
+export enum StringFlags {
+	OTHER_STRING_FLAG = "other-string-flag",
```

Comment:
Should probably remove this for now until we have a flag for it.

Reply:
TypeScript complains when I remove it. I'll just leave it in for future reference.